README.md

```diff
@@ -11,14 +11,14 @@ An Ember CLI addon which allows you to specify CSS inside of component pod direc
 
 ## Usage
 
-This addon allows you to specify a `styles.css` file inside of your component's pod folder (`app/my-component/styles.css`) (the component must also provide a `component.js` returning a valid instance of `Ember.Component`).
+This addon allows you to specify a style sheet inside of your component's pod folder. (the component must also provide a `component.js` returning a valid instance of `Ember.Component`)
 
-Rules defined inside of these `styles.css` files will automatically be namespaced with an autogenerated class. That autogenerated class will also be injected into your component's `classNames` property. This enables you to worry less about rules clashing across component styles.
+Rules defined in the style-sheet will automatically be namespaced with an autogenerated class. That autogenerated class will also be injected into your component's `classNames` property. This enables you to worry less about rules clashing across component styles.
 
 For example, given this `app/my-component/styles.css` file:
 
 ```css
-& {
+& {  // ampersand refers to the component itself (parent selector)
```

Comment:
you can actually use `:--component` also if you want! But you don't have to add that here. I might see if @topaxi can add a little more clarity on how that exactly works in relation to the plugin that he wrote.